packages/core/src/external/carequality/ihe-gateway-v2/saml/saml-client.ts

```diff
@@ -114,6 +119,12 @@ export async function sendSignedXmlMtom({
       "Cache-Control": "no-cache",
     },
     httpsAgent: agent,
+    responseType: "arraybuffer",
```

Comment:
very important. took 2 days to figure out but without this, axios automatically utf-8 encodes data which rsults in pdfs and tiffs getting corrupted

Reply:
we never convert anything to strings now for DRs. Everything is buffers that never get encoded as strings ever!